abis-supplemental/MinterRAMV0.json

```diff
@@ -0,0 +1,1408 @@
+{
```

Comment:
I don't think we should need any non-interface / non-library ABIs in the main subgraph - if something is missing from the interface, definitely feel free to add it to the contracts repo PR and update the interface or library here 👍

Reply:
Thank you! Updated